evm/gateway.js

```diff
@@ -199,17 +188,24 @@ async function processCommand(config, chain, options) {
         }
 
         case 'approveWithBatch': {
-            const { batchID, api } = options;
+            const { batchID } = options;
 
             if (!batchID) {
                 throw new Error('Batch ID is required for the approve action');
             }
 
             const batchId = batchID.startsWith('0x') ? batchID.substring(2) : batchID;
-            let apiUrl = api || `${config.axelar.lcd}/axelar/evm/v1beta1/batched_commands/${chain.name.toLowerCase()}`;
-            apiUrl = apiUrl.endsWith('/') ? apiUrl.slice(0, -1) : apiUrl;
+            let apiUrl = `${config.axelar.lcd}/axelar/evm/v1beta1/batched_commands/${chain.axelarId}/${batchId}`;
 
-            const executeData = await fetchBatchData(apiUrl, batchId);
+            let executeData;
+
+            try {
+                const response = await httpGet(`${apiUrl}`);
+                executeData = '0x' + response?.execute_data;
```

Comment:
missing check on the signing status from the response, to determine if batch is ready to be relayed